src/roce_sq.py

```diff
@@ -524,39 +536,39 @@ def flush(self):
         self.pending_wr_ctx_dict.clear()  # Delete all pending WR
         self.req_pkt_psn_wr_ssn_dict.clear()  # Delete all pending request data
 
-    # oldest_sent_ts_ns is updated in 1 case:
-    # - oldest_sent_ts_ns is None and there's new packet to send
+    # oldest_post_send_or_ack_ts_ns is updated in 1 case:
```

Comment:
We should use a vec or list to store the oldest_ack, for example:

There are two request in flighting and both of them request a ack. If the first request's ack comes, we should leave the second ack waiting info there for timeout checker.

Reply:
done.